CRAMv3.tex

```diff
@@ -920,138 +921,325 @@ \section{\textbf{Record structure}}
 other hand, data series that do not require binary coding can be stored separately 
 in external blocks with some other compression applied to them independently.
 
+As CRAM data series may be interleaved within the same blocks\footnote{Interleaving can sometimes provide better compression, however it also adds dependency between types of data meaning it is not possible to selectively decode one data series if it co-locates with another data series in the same block.} understanding the order in which CRAM data series must be decoded is vital.
+
+The overall flowchart is below, with more detailed description in the subsequence sections.
+
+\algnewcommand\algorithmicto{\text{ \textbf{to} }}
+
 \subsection{\textbf{CRAM record}}
 
 Both mapped and unmapped reads start with the following fields. Please note that 
 the data series type refers to the logical data type and the data series name corresponds 
 to the data series encoding map.
 
-\begin{tabular}{|>{\raggedright}p{36pt}|>{\raggedright}p{70pt}|>{\raggedright}p{75pt}|>{\raggedright}p{90pt}|>{\raggedright}p{171pt}|}
+\begin{tabular}{|>{\raggedright}p{70pt}|>{\raggedright}p{75pt}|>{\raggedright}p{90pt}|>{\raggedright}p{171pt}|}
+\hline
+\textbf{Data series type} & \textbf{Data series name} & \textbf{Field} & \textbf{Description}\tabularnewline
 \hline
- & \textbf{Data series type} & \textbf{Data series name} & \textbf{Field} & \textbf{Description}\tabularnewline
+int & BF & CRAM bit flags & see CRAM record bit flags\tabularnewline
 \hline
-1 & int & BF & CRAM bit flags & see CRAM record bit flags\tabularnewline
+int & CF  & compression bit flags & see compression bit flags\tabularnewline
 \hline
-2 & int & CF  & compression bit flags & see compression bit flags\tabularnewline
+- & - & Positional data & See section \ref{subsec:positions}\tabularnewline
 \hline
-3 & int & RI & ref id & reference sequence id, not used for single reference slices, 
-reserved for future multiref slices. \tabularnewline
+- & - & Read names & See section \ref{subsec:readnames}\tabularnewline
```

Comment:
Reference is `names` below (line 1076) so this does not render

Reply:
Resolving things I've done locally just for ease of tracking.  I'll push an update once I've completed my round of revisions.

Thanks for taking the time to review.